src/commands/run.js

```diff
@@ -242,6 +243,33 @@ export const handler = async (argv) => {
         symbol: '📃'
     })
 
+    if (budget) {
+        status.stopAndPersist({
+            text: 'Asserting against performance budget of config file',
+            symbol: '🧐 '
+        })
+        const preparedBudget = prepareBudgetData(budget)
+        for (const [metric, [{ l: lowerLimit, u: upperLimit }]] of Object.entries(preparedBudget)) {
+            const actual = performanceLog[0].metrics[metric]
+            /**
+             * don't fail if:
+             * - metric is within budget
+             * - we don't care about this metric
+             */
+            if (actual <= upperLimit && actual >= lowerLimit) {
```

Comment:
in this if statement we only check if the metric is within the budget but we don't have a check whether or not we care about it. Did you miss that or is this part of a different check.